Added customer promise responsibilities.md (#35246)
fleetdm/fleet: handbook/customer-success/README.md

3. After the business review, save the presentation as a PDF and share it with your customer.

### Close out completed customer promises

Customer promises are contractually obligated feature requests, with guaranteed completion in specific timeframes. These are always represented in a signed contract with the customer. Document the completion of a customer promise through the following steps:
1. When a customer promise is thought to be complete, Fleet's product team will reach out and ask whoever the assigned CSM is for confirmation.

> 2. If you are notified, please reach out to your customer and schedule a meeting to review the work that has been done, and to make sure it meets their requirements.

Reviewer: ```suggestion
2. Once notified, reach out to your customer and schedule a meeting to review the work that has been done, and to make sure it meets their requirements.
```
